src/main/webapp/script.js

```diff
@@ -231,22 +220,20 @@ function submitItem(omdbItem) {
 }
 
 /**
- * Updates the display of buttons used for pagination if the
- * current page is full.
+ * Fetches for more entertainment items if the current page is fully scrolled
+ * down.
  *
- * @param { number } itemsInPage - number of items loaded in the page
  * @param { string } pageCursor - opaque key representing the cursor for the
  *     next page
  */
-function updatePagination(itemsInPage, pageCursor) {
-  const MAX_ITEMS_PER_PAGE = 18;
-  // Display pagination button only when needed(page is full).
-  if (itemsInPage === MAX_ITEMS_PER_PAGE) {
-    $('#paginationNav').removeClass('d-none');
-    $('#nextLink').attr('href', '/index.html?cursor=' + pageCursor);
-  } else {
-    $('#paginationNav').addClass('d-none');
-  }
+function updatePagination(pageCursor) {
+  $(window).off().scroll(function() {
+    if ((window.innerHeight + window.scrollY) >= document.body.offsetHeight) {
+      $(window).off('scroll');
+
+      getDashboardItems(false, pageCursor);
```

Comment:
tiny nit (usually for javadoc, but I think it can apply here too)

/* shouldRefresh= */ false

Reply:
OK I added comment indicating the meaning of the boolean value